app/poros/board_game.rb

```diff
@@ -29,4 +31,17 @@ def num_players
       "#{@min_players}-#{@max_players}"
     end
   end
+
+  def game_type
+    @game_type_ids.each do |game_category|
+      @available_categories.each do |category|
+        if game_category[:id] == category.id
+          @game_type << category.name
+        end
+      end
+    end
+    return nil if @game_type.empty?
```

Comment:
I know during stand up this was returning an empty string, but I thought I should be more uniform. I can change this if need be.